server/src/models/users.model.ts

```diff
@@ -2,15 +2,11 @@ import { Sequelize, DataTypes, Model, ModelCtor } from 'sequelize'
 
 import { Tag } from './tags.model'
 import { IMinimatch } from 'minimatch'
+import { UserBaseDto } from '../../../shared/types'
 
 const USER_MODEL_NAME = 'user'
 
-export interface User extends Model {
-  id: string
-  username: string
-  displayname: string
-  views: number
-}
+export interface User extends Model, UserBaseDto {}
```

Comment:
could you leave a comment here to the effect of deprecating the use of Model and ModelCtor in favour of ModelDefined? The idea is that in time, we don't extend Model, instead wrapping around ModelDefined. This would also allow us to use the name User to mean the plain old object, rather than suffixing with Dto.

Reply:
addressed in https://github.com/opengovsg/askgovsg/pull/216/commits/5f0c96e1efd5dfeb9a0f6d21a84b27d7d9688995